### ISSUE TITLE
Integration with NBO

### ISSUE THREAD

**Issue by DitchingIt** [obophenotype/zebrafish-phenotype-ontology#49]
Hi. My understanding from ZFIN is that they won't integrate with NBO but are staying with the levels of behavioural process provided by integrating with GO. If so, I am puzzled to find a single example of an NBO term (and its parents) in the ZP ontology: ‘chewing’ (NBO:0000073). I wonder why, not least because the ZP ontology also contains its synonym 'mastication' (GO:0071626).

**Comment by matentzn** [comment]
@DitchingIt I just double checked, I do not see a _ZP_ id associated with an NBO term - so its probably coming in from an imported ontology. How exactly are you using ZP? If you just need a simple vocabulary of the zebrafish phenotypes, maybe you would prefer a release that just contains those (excluding the imported terms)?

**Comment by DitchingIt** [comment]
@matentzn Appreciated and apologies - I am new to the structure of ontologies (and GitHub), but [this ](https://www.ebi.ac.uk/ols/ontologies/zp)is where I saw the item in question under entity, occurrent, process, biological_process, response to stimulus, behavior, behavior process... I realise now from your comment that entity is a sibling of ZP, although I am not clear:

1. where the siblings come from in the link I have given, nor
2. why there is only the one class (chewing) at the end of the behavior process branch. 

I don't need a zebrafish-specific vocabulary release by the way, but thanks for offering. Hoping to engage more in the NBO discussion.

**Comment by matentzn** [comment]
Yes, this is, unfortunately, a common misconception in all OBO ontologies - the presence of a term may simply indicate that it is needed by some other logical definition the is needed by ZP. I don't know how that specific term made its way in though. 

I will close the ticket now, but feel free to open a new one (also on the NBO tracker) if you want to know more. Even better: Join the OBO slack! http://obofoundry.org/
